docs/CONTRIBUTORS

```diff
@@ -76,3 +76,4 @@ Yossi Nivin
 Yuri Victorovich
 Yutetsu TAKATSUKASA
 Yves Mathieu
+Ryszard Różak
```

Comment:
Names in `CONTRIBUTORS` should be listed in alphabetical order.

Reply:
I didn't know about it. I fixed it, thank you.